src/binary-reader-objdump.cc

```diff
@@ -296,6 +297,16 @@ Result BinaryReaderObjdumpDisassemble::OnOpcodeBare() {
   return Result::Ok;
 }
 
+Result BinaryReaderObjdumpDisassemble::OnOpcodeIndex(Index value) {
```

Comment:
I think I will rename these OnOpcode* callback in a follow up to make this more clear